drape_frontend/stylist.cpp

```diff
@@ -434,6 +430,18 @@ bool InitStylist(FeatureType & f, int8_t deviceLang, int const zoomLevel, bool b
 
   CaptionDescription & descr = s.GetCaptionDescriptionImpl();
   descr.Init(f, deviceLang, zoomLevel, geomType, aggregator.m_mainTextType, aggregator.m_auxCaptionFound);
+  // We want minimal displacement priority for house numbers and it
+  // can not be set via drules because usually house numbers are just
+  // an attribute of a feature with some other types.
+  if (descr.IsHouseNumberInMainText() || descr.IsHouseNumberInAuxText())
```

Comment:
1. Won't it slow down the rendering?
2. Cache bools.

Reply:
They're calculated once. Its just getting a stored bool value from the object.